frontend/src/components/Icons.tsx

```diff
@@ -85,19 +114,189 @@ export const NewTabIcon = (
   );
 };
 
-export const Cogwheel = () => (
+/** Icon to indicate there is a menu */
+export const MenuIcon = ({
+  alt,
+  ...props
+}: SVGProps<SVGSVGElement> & { alt: string }) => {
+  return (
+    <svg
+      width={20}
+      height={20}
+      viewBox="0 0 20 20"
+      role="presentation"
+      xmlns="http://www.w3.org/2000/svg"
+      style={{
+        fill: "currentcolor",
+        ...props.style,
+      }}
+      {...props}
+    >
+      {alt && <title>{alt}</title>}
```

Comment:
Ah, I recognise this issue! I think Axe was complaining about an empty `<title>`, no? I solved this with an `aria-hidden` (only applied when `alt === ""`). I think the actual fix you did here is the `role="presentation"`, which I think is even better - but could you make sure that's only applied if `alt === ""`? Otherwise if an alt text _is_ specified, it still won't be read out by screen readers. Likewise for the other instances in this file.

And then also restoring the `aria-label` - IIRC you needed both that and `<title>` [for inline SVG's alt text](https://stackoverflow.com/a/4756461).

(I think the `alt &&` then shouldn't be needed, but it doesn't really do any harm either, so :shrug: )

Reply:
I set it back to `img` - makes the most sense since we want to use `aria-label` and `aria-hidden` - if we use `presentation` we have to remove those global aria properties. I went ahead and made that change to all the icons too - in makes sense in hindsight.